Fallback to platform default ViewType if it's unspecified

```
Caused by: java.lang.NullPointerException: Cannot invoke "me.devnatan.inventoryframework.ViewType.normalize(int)" because the return value of "me.devnatan.inventoryframework.ViewConfig.getType()" is null
        at me.devnatan.inventoryframework.pipeline.OpenInterceptor.createRenderContext(OpenInterceptor.java:74) ~[?:?]
        at me.devnatan.inventoryframework.pipeline.OpenInterceptor.finishOpen(OpenInterceptor.java:58) ~[?:?]
        at me.devnatan.inventoryframework.pipeline.OpenInterceptor.intercept(OpenInterceptor.java:33) ~[?:?]
        at me.devnatan.inventoryframework.pipeline.OpenInterceptor.intercept(OpenInterceptor.java:16) ~[?:?]
```

--- a/platform/src/main/java/me/devnatan/inventoryframework/PlatformView.java
+++ b/platform/src/main/java/me/devnatan/inventoryframework/PlatformView.java
@@ -63,7 +63,7 @@ public abstract class PlatformView<
      */
     @NotNull
     public final ViewConfigBuilder createConfig() {
-        return new ViewConfigBuilder();
+        return new ViewConfigBuilder().type(ViewType.CHEST);
     }
 
     /**

